### PR TITLE
Feature/add new use model

### PR DESCRIPTION
**Pull request by boba-and-beer**

### CODE REVIEW

**Comment by JackyKoh** [comment]
add the other base version:
https://tfhub.dev/google/universal-sentence-encoder-cmlm/en-base/1

and the multilingual version:
https://tfhub.dev/google/universal-sentence-encoder-cmlm/multilingual-base/1
https://tfhub.dev/google/universal-sentence-encoder-cmlm/multilingual-base-br/1

https://tfhub.dev/google/universal-sentence-encoder-cmlm/multilingual-preprocess/2

**Comment by boba-and-beer** [comment]
I have raised an error with the Multilingual models which can be tracked here: https://github.com/tensorflow/hub/issues/732

File: vectorhub/bi_encoders/text_image/torch/clip.py [hunk 1]
```diff
@@ -20,6 +20,10 @@
 
 class Clip2Vec(BaseImage2Vec, BaseText2Vec):
     definition = CLIPModelDefinition
+    urls = {
+        "ViT-B/32": {'vector_length': 512},
+        "RN50": {'vector_length': {512}}
```
Comment (JackyKoh) [inline vectorhub/bi_encoders/text_image/torch/clip.py:25]: {512} - >512

Comment (boba-and-beer) [reply vectorhub/bi_encoders/text_image/torch/clip.py]: Fixed in latest push

**Pull request merged**